DESCRIPTION

```diff
@@ -31,21 +32,28 @@ Imports:
     yaml,
     here,
     crayon,
+    callr,
     progress,
+    pingr,
     forcats,
     assertthat,
     rstudioapi,
     stringr,
-    shiny,
+    shiny (>= 1.6.0),
     shinydashboard,
+    shinydashboardPlus (>= 2.0.0),
+    shinyWidgets,
     DT,
     knitr,
     readr,
     anytime
 Suggests: 
     testthat (>= 3.0.0),
     withr (>= 2.5.0),
-    pillar
+    devtools,
```

Comment:
Related to comment above, what are these new packages needed for? If they're needed for some function (like, I think `httpuv` is [needed for `random_dynamic_port()`](https://github.com/metrumresearchgroup/mrgda/blob/fd49ff7c7bc8b148afc27b4b48adfb9267e65d4f/R/run-app-bg.R#L135)) then we should gate those too, as mentioned in above comment.

If they're just needed for development (like, I think `devtools()`) then you can remove them from DESCRIPTION and just put them in the `pkgr.yml` so that developers have them.

Reply:
`devtools` _was_ actually needed here for the background tests to run. R CMD Check would fail without it being included in the description. I refactored these tests a bit though, so this might actually be able to be removed now.

Reply:
FWIW `rvest` is needed for testing purposes. They will still need to be included in `Suggests` for ever run those tests in `R CMD Check`, but we could have a procedure for running `devtools::test()` manually for each PR, and make use of an [environment var](https://github.com/metrumresearchgroup/mpn.scorecard/blob/main/tests/testthat/setup.R#L257-L261) for skipping these tests. 

I dont know if that's a good approach though. Needing to remember to run `devtools::test()` for _quite a few_ tests seems prone to human error. I would think keeping them in `Suggests` is the better approach here (and then still use `requireNamespace`)

Reply:
@seth127 Sorry for my lack of clarity - `devtools::load_all()` is called [here](https://github.com/metrumresearchgroup/mrgda/blob/c365c60e78e4acab89df61201d76dd66481ac102/R/v.R#L103-L108), but the option is _set_ in `run_app_bg`. That is the location that triggers the `R CMD Check` failure. But yeah, it fails locally still unfortunately